cmd/libsnap-confine-private/cgroup-support-test.c

```diff
@@ -53,7 +53,8 @@ static void cgroupv2_is_tracking_tear_down(cgroupv2_is_tracking_fixture *fixture
     GError *err = NULL;
 
     sc_set_self_cgroup_path("/proc/self/cgroup");
-    g_remove(fixture->self_cgroup);
+    /* file mocked file may have been removed by the test */
```

Comment:
nitpick: there's one "file" too many in the comment.

Reply:
Thanks for spotting!